tool/planet/constants.go

```diff
@@ -259,6 +259,9 @@ const (
 	// EnvPlanetSELinux is an environment variable that indicates whether
 	// SELinux support has been requested
 	EnvPlanetSELinux = "PLANET_SELINUX"
+	// EnvCriticalNamespaces lists the Kubernetes namespaces to search for
+	// critical system pods.
+	EnvCriticalNamespaces = "PLANET_CRITICAL_NAMESPACES"
```

Comment:
nit: I know the environment variable is for internal consumption mostly, but maybe something like `PLANET_MONITORED_NAMESPACES` should relay the purpose better.